src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -230,22 +231,20 @@ public CatalogObjectMetadataList create(
         }
     }
 
```

Comment:
Could you also change the method 
ResponseEntity<List<CatalogObjectMetadata>> list
to <List<CatalogObjectMetadata>> list